release-repo-scripts/bump_dependency.bash

```diff
@@ -371,7 +371,7 @@ for ((i = 0; i < "${#LIBRARIES[@]}"; i++)); do
 
   # libN
   sed -i -E "s ((${LIB#"ign-"}))${PREV_VER} \1${VER} g" $FORMULA
-  sed -i -E "s ((${LIB_#"ign-"}))${PREV_VER} \1${VER} g" $FORMULA
+  sed -i -E "s ((${LIB_#"ign_"}))${PREV_VER} \1${VER} g" $FORMULA
```

Comment:
Cool, it fixes it, but I wonder if there are other problems caused by this. I think `LIB` right now would be something like `ign-gazebo`, and `LIB_` is `ign_fuel_tools`, but the equivalent should be `ign-fuel_tools`.

Reply:
in the case of fuel tools, `LIB` is `ign-fuel-tools` and `LIB_` is `ign_fuel_tools`. The `#"ign-"` and `#"ign_"` commands strip the prefix so that `fuel-toolsX` and `fuel_toolsX` are the strings that are used for search.

Without this change, the first `sed` command was replacing `fuel-tools7` with `fuel-tools8`, while the second command was trying to replace `ign_fuel_tools7` with `ign_fuel_tools8`, which isn't as useful